en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md

```diff
@@ -0,0 +1,725 @@
+            <td>The ID of the customer whose ads are being modified. Type: string</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>operations</code></td>
+            <td>The list of operations to perform on individual ads. Type: object [AdOperation](https://developers.google.com/google-ads/api/rest/reference/rest/v17/AdOperation)</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>partialFailure</code></td>
+            <td>If true, successful operations will be carried out and invalid operations will return errors. If false, all operations will be carried out in one transaction if and only if they are all valid. Default is false. Type: boolean</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td><code>validateOnly</code></td>
+            <td>If true, the request is validated but not executed. Only errors are returned, not results. Type: boolean</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td><code>responseContentType</code></td>
+            <td>The response content type setting. Determines whether the mutable resource or just the resource name should be returned post mutation. Type: enum [ResponseContentType](https://developers.google.com/google-ads/api/rest/reference/rest/v17/ResponseContentType)</td>
+            <td>No</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+
+    ```xml
+    <googleAds.adsMutate configKey="GGOGLE_ADS_CONN">
+        <customerId>{json-eval($.customer_id)}</customerId>
+        <operations>{json-eval($.operations)}</operations>
+    </googleAds.adsMutate>
+    ```
+ 
+    **Sample request**
+
+    ```json
+    {
+        "customer_id": "123123123",
+        "operations": [
+            {
+                "updateMask": "finalUrls",
+                "update": {
+                    "resourceName": "customers/123123123/ads/789789789",
+                    "finalUrls": [
+                        "https://www.github.com"
+                    ]
+                }
+            }
+        ]
+    }
+    ```
+
+??? note "search"
+    The `search` operation returns all rows that match the search query.
+    <table>
+        <tr>
+            <th>Parameter name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td><code>customerId</code></td>
+            <td>The ID of the customer being queried. Type: string</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>query</code></td>
+            <td>The query string. Type: string</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>pageToken</code></td>
+            <td>Token of the page to retrieve. If not specified, the first page of results will be returned. Use the value obtained fromnextPageTokenin the previous response in order to request the next page of results. Type: string</td>
```

Comment:
Please fix the words `fromnextPageTokenin`.

Reply:
Fixed with [70b7f97](https://github.com/wso2/docs-mi/pull/933/commits/70b7f970e86199fba8829404016b58ac512ab3a9)